# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1294638312 28800
#      Sun Jan 09 21:45:12 2011 -0800
# Node ID 44032aac5223795aee4463e197b4d151eaa87d6d
# Parent  1740012184f99e5f2e127a8d23228d5025b83aab
Correct failing error() tests due to change in capitalization of previous changeset.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,14 @@
+2011-01-09  Rik  <octave@nomad.inbox5.com>
+
+	* plot/orient.m, statistics/tests/kolmogorov_smirnov_test.m,
+	strings/strtrunc.m: Correct failing tests due to change in
+	capitalization of error strings from previous changeset.
+
 2011-01-09  Rik  <octave@nomad.inbox5.com>
 
 	* audio/lin2mu.m, audio/loadaudio.m, audio/mu2lin.m, audio/saveaudio.m,
 	audio/wavwrite.m, deprecated/split.m, deprecated/values.m,
 	general/accumarray.m, general/bitcmp.m, general/cell2mat.m,
 	general/circshift.m, general/curl.m, general/divergence.m,
 	general/genvarname.m, general/interpft.m, general/interpn.m,
 	general/nextpow2.m, general/polyarea.m, general/repmat.m,
diff --git a/scripts/plot/orient.m b/scripts/plot/orient.m
--- a/scripts/plot/orient.m
+++ b/scripts/plot/orient.m
@@ -95,15 +95,15 @@ endfunction
 %!  assert (get (hfig, "papersize"), papersize)
 %!  assert (get (hfig, "paperposition"), paperposition)
 %!test
 %!  orient landscape
 %!  orient tall
 %!  assert (orient, "portrait")
 %!  assert (get (hfig, "papersize"), papersize)
 %!  assert (get (hfig, "paperposition"), tallpaperposition)
-%!fail ("orient ('nobody')", "unknown orientation")
+%!fail ("orient ('nobody')", "unknown ORIENTATION")
 %!test
 %!  orient portrait # errors don't change the state
 %!  assert (orient, "portrait")
 %!  assert (get (hfig, "papersize"), papersize)
 %!  assert (get (hfig, "paperposition"), tallpaperposition)
 %!  close (hfig)
diff --git a/scripts/statistics/tests/kolmogorov_smirnov_test.m b/scripts/statistics/tests/kolmogorov_smirnov_test.m
--- a/scripts/statistics/tests/kolmogorov_smirnov_test.m
+++ b/scripts/statistics/tests/kolmogorov_smirnov_test.m
@@ -108,17 +108,17 @@ function [pval, ks] = kolmogorov_smirnov
   if (nargout == 0)
     printf ("pval: %g\n", pval);
   endif
 
 endfunction
 
 %!error <Invalid call to kolmogorov_smirnov_test>
 %!  kolmogorov_smirnov_test (1);
-%!error <kolmogorov_smirnov_test: x must be a vector>
+%!error <kolmogorov_smirnov_test: X must be a vector>
 %!  kolmogorov_smirnov_test ({}, "unif", 2, 4);
 %!error <kolmogorov_smirnov_test: no not_a_distcdf or not_a_dist_cdf function found>
 %!  kolmogorov_smirnov_test (1, "not_a_dist");
 %!error <kolmogorov_smirnov_test: alternative bla not recognized>
 %!  kolmogorov_smirnov_test (1, "unif", 2, 4, "bla");
 %!test # for recognition of unifcdf function
 %!  assert (kolmogorov_smirnov_test (0:100, "unif", 0, 100), 1.0, eps);
 %!test # for recognition of logistic_cdf function
diff --git a/scripts/strings/strtrunc.m b/scripts/strings/strtrunc.m
--- a/scripts/strings/strtrunc.m
+++ b/scripts/strings/strtrunc.m
@@ -48,12 +48,12 @@ function s = strtrunc (s, n)
 
   if (s_was_char)
     s = s{:};
   endif
 
 endfunction
 
 %!error <Invalid call to strtrunc> strtrunc ();
-%!error <s must be a character string or a cell array of strings> strtrunc (1, 1)
+%!error <S must be a character string or a cell array of strings> strtrunc (1, 1)
 %!assert (strtrunc("abcdefg", 4), "abcd");
 %!assert (strtrunc("abcdefg", 10), "abcdefg");
 %!assert (strtrunc({"abcdef", "fedcba"}, 3), {"abc", "fed"});
